engines/wasmtime/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.48
+FROM rust:1.51
```

Comment:
FWIW, this won't be available until https://github.com/docker-library/official-images/pull/9856 merges.